ansible_base/rbac/models.py

```diff
@@ -330,8 +336,8 @@ class Meta:
         abstract = True
 
     # role_definition set on child models to set appropriate help_text and related_name
-    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE)
-    object_id = models.TextField(null=False)
+    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE, help_text=_("The content type this object role applies to."))
+    object_id = models.TextField(null=False, help_text=_("The object id this role applies to."))
```

Comment:
You're actually going to be wrong, unexpectedly, here.

This is an abstract model used for several down the chain, notably assignments. This is written for it being on a Role-like model. But that's actually... not, for what matters. The effect we care most about would apply this to _assignment_ models.

These _will_ be user-facing.

Unfortunately, I think that means you need to be intentionally vague.

Say something like

> The content type of the subject of permission assignments. Duplicated from related RoleDefinition.

> The database primary key of the subject of permission assignments.

And we could go on like a lawyer about the definition of "subject". If "John has permission to pet dog Fido", then that looks like "_Actor_ has permission to _action_ (_content-type_ _identifier_) --> (_subject_)"